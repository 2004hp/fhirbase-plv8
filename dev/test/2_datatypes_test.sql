--db:fhirb
--{{{
select * from fhir.datatype_unified_elements;
select * from fhir.enums;
--}}}
