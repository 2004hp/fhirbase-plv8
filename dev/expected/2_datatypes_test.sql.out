select count(*) from fhir.datatype_unified_elements;
 count
-------
   124
(1 row)

select count(*) from fhir.enums;
 count
-------
    13
(1 row)

